.github/ISSUE_TEMPLATE/spec_issue.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification release
+title: ''
+label: Release Review
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the  with the content defined as follows.
```

Comment:
Missing some words...  Maybe something like this:  "..create issues in the [MicroProfile-WG repository](https://github.com/microprofile/microprofile-wg) with.."